packages/edit-site/src/components/create-template-part-modal/index.js

```diff
@@ -39,6 +54,49 @@ export default function CreateTemplatePartModal( { closeModal, onCreate } ) {
 		[]
 	);
 
+	async function createTemplatePart() {
+		if ( ! title ) {
+			createErrorNotice( __( 'Title is not defined.' ), {
```

Comment:
Similar suggestion here:
```suggestion
			createErrorNotice( __( 'Please enter a title.' ), {
```

Reply:
This notice was the same as the existing create template modal. Shall I update that one as well?

Reply:
Ok, I think leave it for now, and we can try to address it in a separate PR and get a design review on that.